### PR TITLE
Clear cache on url rewriting settings modification

### PR DESCRIPTION
**Pull request by antoin-m**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | The cache should be cleared when enabling/disabling friendly URL rewriting
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1825

How to test? 
<ul><li>FO: Go on the front page (in order to cache the menu)</li><li>BO: Disable friendly URLs (Shop parameters => Traffic)</li><li>FO: Reload the front page</li><li>FO: Items of the menu should not 404</li></ul>

### CODE REVIEW

**Comment by aleeks** [comment]
@antoin-m , please, rebase when https://github.com/PrestaShop/PrestaShop/pull/6912 will be merged. Yours PR are goods, so you need to mix them :p

File: classes/Tools.php [hunk 1]
```diff
@@ -2751,6 +2751,20 @@ public static function clearSmartyCache()
...
+    /**
+     * Clear Symfony cache
+     */
+    public static function clearSf2Cache($env = null)
+    {
+        if (!$env) {
+            $env = _PS_MODE_DEV_ ? 'dev' : 'prod';
+        }
+
+        $sf2Refresh = new \PrestaShopBundle\Service\Cache\Refresh($env);
```
Comment (mickaelandrieu) [inline classes/Tools.php]: I'm not sure this is a good idea to do this, wdyt @thierrymarianne  ?

Comment (antoin-m) [inline classes/Tools.php]: As you can see in this pull request it is already done in a lot of places. I just DRYed it.

**Comment by aleeks** [comment]
ping @maximebiloe

**Comment by maximebiloe** [comment]
Thank you @antoin-m

**Pull request merged**